docs/src/tutorial-kinds-of-failure.md

```diff
@@ -45,33 +45,20 @@ But we're able to check this unsafe code with Kani:
 ```
 # kani src/bounds_check.rs --harness bound_check
 [...]
-** 15 of 448 failed
+SUMMARY:
+ ** 1 of 459 failed
 [...]
 VERIFICATION:- FAILED
 ```
 
-Notice there were a *lot* of verification conditions being checked: in the above output, 448! (It may change for you.)
+Notice there were a *lot* of verification conditions being checked in the above output: 459! (This number may change depending on the version.)
```

Comment:
I would probably remove this. We are likely going to hide some of these checks and it may cause confusion if the number goes down.